src/docs/arch-glossary.dox

```diff
@@ -65,8 +63,6 @@ they are used in other settings.
 <tr><td>key prefix compression<td>btree<td>a technique to store identical key prefixes only once per page.  See @ref file_formats_compression
 <tr><td>kv or k/v<td>btree<td>a key/value pair in a Btree.
 <tr><td>leaf page<td>btree<td>either in-memory or on disk, a page in a Btree that has a pairs of keys and values, and no pages beneath it.
-<tr><td>log structured merge tree (or LSM tree)<td>lsm<td>a data structure that stores keys and values (as a Btree), but composed of potentially many trees (Btrees in WiredTiger).
```

Comment:
We could in theory keep this as its a definition, but happy to remove it too.

Reply:
I don't think we need to keep it if the documentation has no references to it. The definition will still be present in older versions of the docs which are accessible.